UPGRADE-4.0.md

```diff
@@ -207,6 +207,16 @@ FrameworkBundle
 
  * Extending `ConstraintValidatorFactory` is not supported anymore.
 
+ * Class parameters in `routing.xml` has been removed
```

Comment:
`has` => `have`. Also I think that the place where parameters were defined does not matter that much for end users. `Class parameters related to routing...`?